client/src/components/Header/Header.jsx

```diff
@@ -1,40 +1,152 @@
 import "./Header.css";
 import { NavLink } from "react-router-dom";
+import { FaBars, FaTimes } from "react-icons/fa";
+import { AiFillCaretUp } from "react-icons/ai";
+import { useState, useEffect } from "react";
 
 const Header = () => {
+	const [isNavOpen, setIsNavOpen] = useState(false);
+	//for closing animation
+	const [isClosingAdded, setIsClosingAdded] = useState(false);
+
```

Comment:
Suggestion: We could do something like this:

```suggestion
	const navLinks = [
		{ id: 1, to: "/", text: "Home" },
		{ id: 2, to: "/about", text: "About LSH" },
		{ id: 3, to: "/services", text: "Services" },
		{ id: 4, to: "/news", text: "News" },
		{ id: 5, to: "/events", text: "Events" },
		{ id: 6, to: "/gallery", text: "Gallery" },
		{ id: 7, to: "/get-involved", text: "Get Involved" },
		{ id: 8, to: "/contact-us", text: "Contact Us" },
	];
```